dashboard/client/src/pages/node/index.tsx

```diff
@@ -101,6 +102,10 @@ const columns = [
   },
   { label: "Sent" },
   { label: "Received" },
+  {
+    label: "Logical Resource",
+    helpInfo: `You can access the logical usage (e.g., CPU, memory) for a node. Check https://docs.ray.io/en/latest/ray-core/scheduling/resources.html#physical-resources-and-logical-resources for more details. Alternatively, you could run the CLI command "ray status -v" to obtain a similar result.`,
```

Comment:
```suggestion
    helpInfo: `The logical usage (e.g., CPU, memory) for a node. Check https://docs.ray.io/en/latest/ray-core/scheduling/resources.html#physical-resources-and-logical-resources for more details. Alternatively, you could run the CLI command "ray status -v" to obtain a similar result.`,
```

Also, can we make the url a link so users can just click it?

Reply:
```suggestion
    helpInfo: `[Logical resource usage](https://docs.ray.io/en/latest/ray-core/scheduling/resources.html#physical-resources-and-logical-resources) (e.g., CPU, memory) for a node. Alternatively, you can run the CLI command `ray status -v` to obtain a similar result.`,
```
- Let's simplify it a bit?
- +1 to use link
- also format the CLI command as code (use font like Roboto mono)?

Reply:
Good idea!

Reply:
@scottsun94 Updated Tooltip
<img width="403" alt="Screenshot 2023-07-26 at 2 46 17 PM" src="https://github.com/ray-project/ray/assets/125417081/88bc3f70-b68e-4e0e-9255-eb85bffff99b">